internal/services/dashboard/dashboard_grafana_resource.go

```diff
@@ -89,6 +107,70 @@ func (r DashboardGrafanaResource) Arguments() map[string]*pluginsdk.Schema {
 			}, false),
 		},
 
+		"configuration": {
+			Type:     pluginsdk.TypeList,
+			Optional: true,
+			MaxItems: 1,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*schema.Schema{
+					"smtp": {
+						Type:     pluginsdk.TypeList,
+						Optional: true,
+						MaxItems: 1,
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*schema.Schema{
+								"smtp_enabled": {
```

Comment:
```suggestion
								"enabled": {
```
smpt is in the block name?